instituto-brasileiro-de-informacao-em-ciencia-e-tecnologia-abnt-initials.csl

```diff
@@ -412,168 +530,303 @@
     </sort>
     <layout>
       <choose>
-        <if type="article article-journal article-magazine article-newspaper" match="any">
-          <group suffix=".">
-            <text macro="author" suffix=". "/>
-            <text macro="title" suffix=". "/>
-            <text macro="translator" suffix=". "/>
-            <text macro="container-title" suffix=", "/>
-            <text variable="collection-title" suffix=". "/>
-            <text macro="place" suffix=", "/>
-            <text macro="locators" suffix=", "/>
-            <text macro="issued" suffix=". "/>
+        <if type="article" match="any">
+          <group delimiter=". " suffix=".">
+            <text macro="author"/>
+            <text macro="title"/>
+            <text macro="translator"/>
+            <text macro="contributor"/>
+            <group delimiter=", ">
+              <text macro="publisher"/>
+              <text macro="locators"/>
+              <text macro="issued"/>
+            </group>
             <text macro="access"/>
           </group>
         </if>
+        <else-if type="article-journal article-magazine article-newspaper" match="any">
+          <group delimiter=". " suffix=".">
+            <text macro="author"/>
+            <text macro="title"/>
+            <text macro="translator"/>
+            <text macro="contributor"/>
+            <group delimiter=", ">
+              <text macro="container-title"/>
+              <text macro="place"/>
+              <text macro="locators"/>
+              <text macro="issued"/>
+            </group>
+            <text macro="access"/>
+          </group>
+        </else-if>
         <else-if type="bill">
-          <group suffix=".">
-            <text macro="author" suffix=". "/>
-            <text macro="title" suffix=". "/>
-            <text variable="abstract" suffix=". "/>
-            <text macro="container-title" suffix=", "/>
-            <text variable="section" prefix="seç. " suffix=", "/>
-            <text variable="volume" prefix="n. " suffix=", "/>
-            <text variable="page" prefix="p. " suffix=", "/>
-            <text macro="issued" suffix="."/>
+          <group delimiter=". " suffix=".">
```

Comment:
All these `suffix="."` that you have set here for each group in each item-type conditional can just be set once on bibliography>layout. Please replace those.